tensorflow_datasets/core/registered.py

```diff
@@ -164,7 +164,8 @@ def builder(name, **builder_init_kwargs):
     DatasetNotFoundError: if `name` is unrecognized.
   """
   name, builder_kwargs = _dataset_name_and_kwargs_from_name_str(name)
-  builder_kwargs.update(builder_init_kwargs)
+  builder_kwargs.update( \
```

Comment:
I may be wrong but I don't think you should not modify `register.py`. Unless generating `tfds.load('scientific_paper:1.1.0')` generate `1.1.1`.

Reply:
`tfds.builder('scientific_paper:1.1.0', version = None)` generates `1.1.1`

Reply:
Just read the documentation.
```
**builder_init_kwargs: `dict` of keyword arguments passed to the
   `DatasetBuilder`. These will override keyword arguments passed in `name`,
    if any.
```
So, I fixed this error in the script.